spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -1,5 +1,32 @@
 describe 'orifice-calculator-viewmodel-spec', ->
-  viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
+  viewModel = null
+  field = ko.observable()
+
+  beforeEach ->
+    viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
+
+  itBehavesLikeMandatoryField = () ->
```

Comment:
field should be the attribute of the method and when called you pass it as an argument.

```
itBehavesLikeMandatoryField = (field) ->